.golangci.yml

```diff
@@ -25,24 +35,46 @@ linters:
   - gosec
   - gosimple
   - govet
+  - importas
   - ineffassign
+  - makezero
   - misspell
   - nakedret
+  - nilnil
+  - nlreturn
```

Comment:
What's the point of this linter? To be honest, it seems a bit annoying and I don't see any advantages.

Reply:
Separated "return" from the rest of the function body increases readability. And since we read the code much more often compared to write, it's crucial to make it as readable as possible. nlreturn just another microstep in that direction.